### PR TITLE
Added Vaadin Directory Badge

### PR DESCRIPTION
**Pull request by binbalenci**
Hello, I am Binh from Vaadin Directory team. I have tested and reviewed your component and I think it would be great to list your component in our Directory.  By adding this super cool badge to your readme, your project going to be automatically published there when we release our Directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![Vaadin Directory availability](https://img.shields.io/badge/Vaadin%20Directory-available-green.svg?style=flat-square&colorB=00b4f0&link=https://vaadin.com/directory)
```
Comment (warpech) [inline README.md:1]: Is this supposed to be a link? It opens the SVG file for me.

**Comment by warpech** [comment]
You are welcome to include this component in your directory, but value does it have for the repo to have this badge in it?

**Comment by alshakero** [comment]
Also, a search shows it actually isn't available on [VD](https://vaadin.com/directory/search?keyword=juicy-select)?

**Comment by warpech** [comment]
Closing for lack of response from the committer

**Comment by binbalenci** [comment]
Hi @warpech,

Sorry for the late reply. I was caught up in some big project. Anyhow, I shall answer your question.

First, sorry for the misleading last sentence in my message, ain't nothing going to be automatically published. So basically a badge is just a badge, and yes, we have already tested your component and we want to include it in our Directory because your component is awesome. ;)

Now let me boast a little bit about Vaadin Directory:
• It is the main place for the Vaadin Community (for around 7 years now)
• It has already been popular with the Java developers (around 150K users at the moment)
• It has around 80k to 100k unique view each month

And what is in that for you? Since Vaadin 10 framework supports web components, our company's vision also keeps pace with the future of PWA, and Vaadin Directory is no exception. By having a badge there we promise:
• Your component can be reached by a large user base
• Not only people can see and test your component, they can also give ratings and feedbacks
• Be a part of the bigger ecosystem

I hope I have answered your question. As a matter of fact, we just officially support custom Github badge now. [Have a look](https://vaadin.com/blog/vaadin-directory-now-welcomes-custom-github-badges).

**Pull request closed without merging**